Packages/com.scene.handle.extensions/Handle Extensions/Editor/Snapping/SceneHandles.Snapping.cs

```diff
@@ -562,17 +589,28 @@ private bool GetPlaneIntersection(Vector2 guiPosition, out Vector3 worldPlanePos
                 var origin = worldSlidePlane.ClosestPointOnPlane(worldSlideOrigin);
                 return GetIntersectionOnAlternativePlane(worldRay, normal, origin, out worldPlanePosition);
             }
-
+            
             if (!worldSlidePlane.Raycast(worldRay, out dist)) { dist = float.PositiveInfinity; }
-            if (dist > camera.farClipPlane)
+
+            float farClipPlaneDistance = camera.farClipPlane * 0.5f;
```

Comment:
Not sure if 0.5f is a good number, but farClipPlane is definitely too far away to place the alternative plane on.
Might want to revisit this solution later on